crates/cli/Cargo.toml

```diff
@@ -9,7 +9,7 @@ anyhow = "1.0.71"
 blake3 = { version = "1.5", features = [ "mmap" ] }
 clap = { version = "4.0.29", features = ["derive"] }
 futures-util = { version = "0.3", default-features = false }
-gevulot-node = { path = "../node" }
+gevulot-node = { path = "../node", default-features = false }
```

Comment:
I extracted this to separate PR as it's needed ASAP for the blog post: https://github.com/gevulotnetwork/gevulot/pull/117

Reply:
OK I see I'll rebase.